app/api/ada/lib/storage/bridge/metadataUtils.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import { RustModule } from '../../cardanoCrypto/rustLoader';
+
+export type TransactionMetadata = {|
+  label: string,
+  data: Object,
```

Comment:
just FYI that `Object` is aliased to `any`. If you want to force it to be a JS object, you can use `{...}` instead